OPi/GPIO.py

```diff
@@ -263,6 +264,23 @@ def _check_configured(channel, direction=None):
         raise RuntimeError("Channel {0} is configured for {1}".format(channel, descr))
 
 
+class PinDoesNotSupportEdgeDetection(Exception):
+    pass
+
+
+def _pin_has_interrupt(pin):
```

Comment:
can this be called `_check_supports_edge_detection()` so that it is named similarly to `_check_configured()` ?